src/content/contributing/style-guide/index.md

```diff
@@ -100,241 +138,8 @@ When introducing an unfamiliar acronym, spell out the full term, and put the acr
-
-_This isn't an easy one, especially for non-native English speakers. If you aren't sure, don't worry. We'll help with any of these._
-
-### Linking to internal pages {#internal-links}
-
-When linking to another page on Ethereum.org, use the relative path over the absolute path. Do not hard-code the language path (i.e. `/en/`) in any links. This maintains consistent functionality across different language versions of the site.
-
-```md
-<!-- Good -->
-
-Read more about [smart contracts](/docs/developers/smart-contracts/)
-
-<!-- Bad -->
-
-Read more about [smart contracts](/en/docs/developers/smart-contracts)
-Read more about [smart contracts](https://ethereum.org/en/docs/developers/smart-contracts)
-```
-
-Please also add a trailing slash to all links. This keeps links consistent and avoids redirects, which hurts site performance.
-
-```md
-<!-- Good -->
-
-Read more about [smart contracts](/docs/developers/smart-contracts/)
-
-<!-- Bad -->
-
-Read more about [smart contracts](/docs/developers/smart-contracts)
-```
-
-### Linking to images {#images}
-
-When adding an image to a page, the image should be downloaded and placed in the same folder as the markdown file. You can reference the image like this:
-
-`![alt text for image](./image.png)`
-
-```md
-<!-- Good -->
-
-![How to mint your NFT](./mintYourNFT.gif)
-
-<!-- Bad -->
-
-![How to mint your NFT](https://cdn-images-1.medium.com/max/2000/0342fj_fsdfs.gif)
-```
-
-This helps us ensure the image will be available.
-
-### Using emojis {#images}
-
-Everyone loves emojis <Emoji text="🥰" size={1} /> To standardize the appearance of all Emojis across browsers, ethereum.org uses an `<Emoji />` React component.
-
-```md
-<--- Good --->
-
-The London Upgrade is live <Emoji text=":heart:" size={1} />
-
-The London Upgrade is live <Emoji text="❤️" size={1} />
-
-<--- Bad --->
-
-The London Upgrade is live ❤️
-```
-
-### Header casing {#header-casing}
-
-This site uses **sentence casing** for header names as a convention. Only the first word and proper nouns are capitalized. This applies to all markdown files on lines that begin with hashes (#).
-
-```md
-<!-- Good -->
-
-## Minting your NFT
-
-### Setting up your wallet
-
-### Get enough ether
-
-<!-- Bad -->
-
-## Minting Your NFT
-
-### Setting Up Your Wallet
-
-### Getting Enough Ether
-```
-
-### Article authors {#authors}
-
-When citing articles from a specific author or organization, use the article's name as a link, followed by a dash, then the author's name italicized.
-
-```md
-<--- Good --->
-
-- [A rollup-centric ethereum roadmap](https://ethereum-magicians.org/t/a-rollup-centric-ethereum-roadmap/4698) — _Vitalik Buterin_
-- [The History of Ethereum Testnets](https://consensys.net/blog/news/the-history-of-ethereum-testnets/) – _ConsenSys_
-
-<--- Bad--->
+[Content standartization](/style-guide/content-standartization/) - Read more about proper usage of terminology and other aspects such as how to properly add an image, atribute etc.
```

Comment:
```suggestion
[Content standardization](/style-guide/content-standardization/) - Read more about proper usage of terminology and other aspects such as how to properly add an image, attribute etc.
```
Should also update the directory name `standartization` --> `standardization`. This would be mroe aligned with our preference for American English.